### DISCUSSION TITLE
REST API POST /uploads

### DISCUSSION THREAD

**Discussion by PranayReddy98** [fossology/fossology#2193] [Q&A] [answered]
I am running Fossology as a docker container.  I am making use of REST API provided by Fossology.
When I upload a project using repository details, uploads POST endpoint is returning correct uploadId till 9 and after that it is returning uploadId as 1 for all the uploads.

Can someone confirm if they have faced the similar problem? If any of you know solution to the above mentioned problem, please mention it.

**Comment by GMishx** [comment, marked as answer]
Hello @PranayReddy98 , the issue was actually fixed by https://github.com/fossology/fossology/pull/2174

Can you please try to pull the latest image from Docker Hub and check if it is working for you?

**Reply by PranayReddy98** [reply]
Thank you @GMishx. It is working after I pulled the latest image.
